Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1153,9 +1153,10 @@ bool LoadEventNexus::hasEventMonitors() {
         break;
       }
     }
-    m_file->openData("event_id");
```

Comment:
Why is this deleted? Why are we still calling `closeGroup` a few lines further down, given that we do not open anymore?

Reply:
This is deleted because we added the call to `numEvents` which does the same thing in this case.
The call to `closeGroup` corresponds to `openGroup` on line 1152 not the call to `openData` in `numEvents`.